### DISCUSSION TITLE
VERY difficult to get this working on Docker?

### DISCUSSION THREAD

**Discussion by dojoca** [ghostfolio/ghostfolio#2139] [General]
I've set up lots of containers, but none have been as impossible to get working as Ghostfolio. It looks like a great app but honestly I am pretty close to giving up. 

I've tried following the instructions, and I am admittedly a novice.. but it seems to be stuck on trying to find a .env file, which I can't get it to find no matter what I do. 

Is there a simple docker compose that includes all variables and things to get this up and running? Is there a way to do this via portainer?

**Comment by ups7** [comment]
You can try to follow the steps in the following guide - it is for a Synology NAS so perhaps you can ignore any Synology-specific steps and jump straight to the Docker compose file?
https://mariushosting.com/how-to-install-ghostfolio-on-your-synology-nas/

I have it running with Docker (without Portainer).

**Reply by dojoca** [reply]
Hey there - I just wanted to say a big thank you for linking me to that article. I managed to get things working perfectly by adapting the instructions :) Really appreciate it.

**Comment by ups7** [comment]
There is a list of environment variables here:
https://github.com/ghostfolio/ghostfolio#self-hosting

I hope you are aware that you need to have Redis and PostgreSQL running in addition to Ghostfolio. They have their own environment variables, the previous link should help with that.
